rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rental << self
+    @person = person
+    person.rental << self
+  end
```

Comment:
- [ ] To implement the `has-many side` of the relationship(`a person has many rentals`), method for adding rentals should be present in the `Person class` and `@rentals should be initialized`, something like: 

```ruby
  def initialize(age, name = "Unknown", parent_permission = true)
  ...
  ...
    @rentals = []
  end
```
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
Please kindly implement the changes in your `Person Class` in `person.rb` file.